src_files.yml

```diff
@@ -37,6 +37,7 @@ common_cells_all:
     - src/stream_fork.sv
     - src/stream_intf.sv
     - src/stream_join.sv
+    - src/stream_join_dynamic.sv
```

Comment:
`stream_join` now depends on `stream_join_dynamic`, so the former should go to level 1 (same for `Bender.yml`)